improver_tests/nbhood/square_kernel/test_SquareNeighbourhood.py

```diff
@@ -478,83 +395,75 @@ def test_multiple_times(self):
                 [1.0, 1.0, 1.0, 1.0, 1.0],
             ]
         )
-        cube = set_up_cube(
-            zero_point_indices=((0, 0, 2, 2), (0, 1, 1, 2)),
-            num_time_points=2,
-            num_grid_points=5,
-        )
+
+        cube = self.multi_cube
+        cube.data[0, 2, 2] = 0
+        cube.data[1, 1, 2] = 0
         result = SquareNeighbourhood().run(cube, self.RADIUS)
         self.assertIsInstance(cube, Cube)
-        self.assertArrayAlmostEqual(result.data[0, 0], expected_1)
-        self.assertArrayAlmostEqual(result.data[0, 1], expected_2)
+        self.assertArrayAlmostEqual(result.data[0], expected_1)
+        self.assertArrayAlmostEqual(result.data[1], expected_2)
 
     def test_multiple_times_with_mask(self):
         """Test that the run method produces a cube with correct data when a
         cube containing masked data at multiple time steps is passed in.
         Re-masking is disabled."""
-        cube = set_up_cube(
-            zero_point_indices=((0, 0, 2, 2),), num_time_points=2, num_grid_points=5
-        )
+        cube = self.multi_cube
+
```

Comment:
Again, this is a multi-realization cube, not a multi-time cube.  So please would you update the test name and docstring.

Reply:
Done.